core/src/main/scala/kafka/admin/ZkSecurityMigrator.scala

```diff
@@ -70,98 +72,6 @@ object ZkSecurityMigrator extends Logging {
   val usageMessage = ("ZooKeeper Migration Tool Help. This tool updates the ACLs of "
                       + "znodes as part of the process of setting up ZooKeeper "
                       + "authentication.")
-  val workQueue = new LinkedBlockingQueue[Runnable]
-  val threadExecutionContext = 
-    ExecutionContext.fromExecutor(new ThreadPoolExecutor(1,
-                                                         Runtime.getRuntime().availableProcessors(),
-                                                         5000,
-                                                         TimeUnit.MILLISECONDS,
-                                                         workQueue))
-  var futures: List[Future[String]] = List()
-  var exception: ZkException = null
-  var zkUtils: ZkUtils = null
```

Comment:
By creating a class that takes `ZkUtils`, we don't need to worry if `ZkUtils` is assigned at the right place or not.